static/versionDetails.json

```diff
@@ -560,7 +560,7 @@
     },
     {
       "flag": "rookShouldUseAllNodes",
-      "description": "This will disable management of nodes in the CephCluster resource. If false, ekco will add nodes to the storage list and remove them when a node is purged.",
+      "description": "This will disable management of nodes in the CephCluster resource and allow Rook to use all nodes as storage nodes.",
```

Comment:
```suggestion
      "description": "Eanbling this will configure Rook to use all nodes for storage rather than EKCO managing a list of Rook nodes based on labels.",
```

Reply:
This is hard what do you think about this, is that accurate?

Reply:
It is accurate but "managing a list of Rook nodes based on labels" is not a feature we have exposed to kurl. I'm going to dig into ekco functionality a bit more to see what happens exactly if you disable this.

Reply:
Ok I wasn't 100% sure how EKCO managed the node list either, I thought it was by labels but double checking that is absolutely worthwhile.